packages/typegpu/tests/builtin.test.ts

```diff
@@ -16,4 +22,30 @@ describe('builtin', () => {
     expect(resolutionCtx.resolve(code)).toContain('position');
     expect(resolutionCtx.resolve(code)).toContain('front_facing');
   });
+
+  it('is a hybrid type of symbol and a wgsl type', () => {
+    const x = builtin.position.x;
+    const y = builtin.position.y;
+    const z = builtin.position.z;
+
+    // @ts-expect-error
+    const p = builtin.position.p;
+  });
+
+  it('can be omitted from a record type', () => {
+    const x = {
+      a: u32,
+      b: builtin.localInvocationId,
+      c: f32,
+      d: builtin.localInvocationIndex,
+    };
+
+    type X = typeof x;
+    type Omitted = OmitSymbols<X>;
+
+    expectTypeOf(x as Omitted).toEqualTypeOf({
+      a: u32,
+      c: f32,
+    });
```

Comment:
The generic parameter can be used to test a specific type, not the type of a value.
```suggestion
    expectTypeOf<Omitted>().toEqualTypeOf({
      a: u32,
      c: f32,
    });
```